src/mesh_analysis.rs

```diff
@@ -653,4 +720,79 @@ mod tests {
         );
         assert_eq!(genus, 3);
     }
+
+    #[test]
+    fn test_border_edge_loops_should_be_one() {
```

Comment:
Let's name this a bit more explicitely (if you see this test failing in the console, the name doesn't tell you anything). I suggest: `test_border_edge_loops_returns_one_for_xyz`. Same for other test names.